tests/libkvconfig/kvconfig.go

```diff
@@ -56,3 +61,16 @@ func patchKV(namespace, name string, patchOps []patch.PatchOperation) error {
 	_, err = kubevirt.Client().KubeVirt(namespace).Patch(context.Background(), name, types.JSONPatchType, patchData, metav1.PatchOptions{})
 	return err
 }
+
+func PatchWorkloadUpdateMethodAndRolloutStrategy(kvName string, virtClient kubecli.KubevirtClient, updateStrategy *v1.KubeVirtWorkloadUpdateStrategy, rolloutStrategy *v1.VMRolloutStrategy, fgs []string) {
```

Comment:
not related to this PR, just a future enhancement: 
there is a quite similar private function in the `hotplug` pkg.
It will be great to merge them

Reply:
Yes, this was my idea, but I didn't want to do it in this PR